src/ansys/fluent/core/services/interceptors.py

```diff
@@ -29,9 +31,15 @@ def _intercept_call(
         )
         response = continuation(client_call_details, request)
         if not response.exception():
-            network_logger.debug(
-                f"GRPC_TRACE: response = {MessageToDict(response.result())}"
-            )
+            response_str = str(MessageToDict(response.result()))
```

Comment:
I think using [ByteSize()](https://googleapis.dev/python/protobuf/latest/google/protobuf/message.html#google.protobuf.message.Message.ByteSize) method of protobuf message is better than converting it to string.

Reply:
Thanks for letting me know about that method, I wasn't aware of it. 

The main reason why I did not look for an alternate method to get the size, is because we have to serialize the message result into a string regardless, so just getting the size of the Python string directly made sense to me. 

I will try using ByteSize anyway to see what it behaves like.

Reply:
Using ByteSize is indeed better, avoids working with huge strings unless we actually want to log them, using it now, also optimized the logic a little bit